src/test/java/org/llorllale/cactoos/matchers/RunsInThreadsTest.java

```diff
@@ -83,8 +82,12 @@ void matchNegative() {
         ).affirm();
         new Assertion<>(
             "counter must not be incremented by all threads",
-            counter.get(),
-            new IsNot<>(new IsEqual<>(threads * attempts))
+            new IsEqual<>(counter.get()),
+            new Mismatches<>(
+                threads * attempts,
+                "<100>",
+                "was <2000>"
+            )
```

Comment:
@baudoliver7 it looks like to me that here, this would be equivalent to testing the `IsEqual` matcher, which makes no sense, since this is not the purpose of this class. The actual test is the assertion above, so maybe the best here, is to replace this with testing that `counter.get() < threads * attempts`.

Reply:
@victornoel Totally right !